index.bs

```diff
@@ -6901,6 +6901,37 @@ Dictionary {{DelayOptions}} Members</h5>
 	:: The maximum delay time for the node.  See {{BaseAudioContext/createDelay(maxDelayTime)/maxDelayTime!!argument|createDelay(maxDelayTime)}} for constraints.
 </dl>
 
+<h4 id="DelayNode-processing">Processing</h4>
+
+A {{DelayNode}} has an internal buffer that holds {{DelayNode|delayTime}}
+seconds of audio.
+
+The processing of a {{DelayNode}} is broken down in two parts: writing to the
+delay line, and reading from the delay line. This is done via two internal
+{{AudioNode}}s (that are not accessible to content and exist only to ease
```

Comment:
Can you clarify what "accessible to content" means?

Reply:
Sorry this is a Mozilla-ism it seems. I meant "accessible to authors in a web page" or something.